### PR TITLE
Fix broken links

### LINKED ISSUE #284: Fix broken links
https://github.com/web3well/bls-wallet/tree/main/docs

### PR DESCRIPTION
**Pull request by voltrevo**
## What is this PR doing?

Fixes some broken links.

## How can these changes be manually tested?

Check the links go to their intended destination on github.

## Does this PR resolve or contribute to any issues?

Resolves #284.

## Checklist
- [x] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-- [See an overview of BLS Wallet & how the components work together](./docs/system_overview.md)
-- [Use BLS Wallet in a browser/NodeJS/Deno app](./docs/use_bls_wallet_clients.md)
+- [See an overview of BLS Wallet & how the components work together](./system_overview.md)
+- [Use BLS Wallet in a browser/NodeJS/Deno app](./use_bls_wallet_clients.md)
 - Setup the BLS Wallet components for:
-  - [Local develeopment](./docs/local_development.md)
-  - [Remote development](./docs/remote_development.md)
\ No newline at end of file
+  - [Local develeopment](./local_development.md)
+  - [Remote development](./remote_development.md)
```
Comment (jzaki) [inline docs/README.md:7]: I noticed this section was duplicated from `./README.md` (or vice versa).
Do we need a root level readme, and this partial copy in docs/readme?

Comment (jacque006) [reply docs/README.md:7]: This is a duplicate of what we have in the [top level README](https://github.com/web3well/bls-wallet/blob/main/README.md#getting-started) to give context over what is included in current docs. I just messed up my relative pathing when I copy pasta'd.

Open to removing it from the `docs` README if you think it is redundant.

**Review by jacque006 — APPROVED** [review]
Barring addition feedback from @jzaki , this is good to go. Thanks for the fix!

**Comment by jacque006** [comment]
@jzaki If you think we should remove this redundant index in `docs` dir, let me know.

**Pull request merged**